src/test/groovy/com/calgaryscientific/gradle/VeracodeMitigationInfoTest.groovy

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
```

Comment:
what's the different between VeracodeMitigationInfo and VeraCodeGetMitigationInfo?

Reply:
The one is the domain object. The other is the gradle task.

Reply:
then it sounds like you could put Task at the end of the gradleTask to be more explicit about it :)

Reply:
The actual classes are more descriptive:

VeracodeGetMitigationInfoTask.groovy
VeracodeUpdateMitigationInfoTask.groovy
VeracodeMitigationInfo.groovy

It is with the tests that we have a problem:

VeracodeGetMitigationInfoTest.groovy
VeracodeMitigationInfoTest.groovy
VeracodeUpdateMitigationInfoTest.groovy

Maybe I can separate the Domain Object classes and the tasks in separate folders in the future.

Reply:
yes, .... good idea ....